integration/simulation/network/network_utils.go

```diff
@@ -27,6 +27,8 @@ import (
 const (
 	Localhost                  = "127.0.0.1"
 	DefaultWsPortOffset        = 100 // The default offset between a Geth node's port and websocket ports.
+	DefaultHostP2pOffset       = 200 //  The default offset for the host P2p
+	DefaultHostRPCOffset       = 400 //  The default offset for the host RPC
```

Comment:
@tudor-malene Can you use this constant in `simulation/network/socket.go`, on line 125, as well?

Reply:
good point.